lib/msal-browser/src/naa/TokenResponse.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AccountInfo } from "./AccountInfo";
+
+export type TokenResponse = {
+    access_token: string;
+    account: AccountInfo;
+    client_info: string;
+    expires_in: number;
+    id_token: string;
+    properties: TokenResponseProperties | null;
+    scope: string;
+    state: string;
+    shr?: string; // token binding enabled at native layer it is the access token, not the signing keys
+    extendedLifetimeToken?: boolean;
+};
+
+export type TokenResponseProperties = {
+    MATS?: string;
```

Comment:
We have a MATS type defined if you want to re-use that

Reply:
Owned by bridge.